src/runtime/runtime.js

```diff
@@ -816,6 +824,23 @@ export class ConfiguredRuntime {
     this.buttonApi_.attach(button, optionsOrCallback, opt_callback);
   }
 
+  /** @override */
+  renderSmartButton(container, options, callback) {
+    if (!options) {
+      Object.assign({}, {theme: Theme.LIGHT, lang: 'en'});
+    }
+    if (!isExperimentOn(this.win_, ExperimentFlags.SMARTBOX)) {
+      throw new Error('Not yet launched!');
+    }
+    if (!container) {
+      throw new Error('No element found to render Smart button!');
+    }
+    return this.documentParsed_.then(() => {
+      return new SmartSubscriptionButtonApi(this, container, options, callback)
```

Comment:
Is this a good idea to return in an API?

Reply:
Since now it is an API, return is not required.  Removed.